controllers/providers/k8gbendpoint/applicationDNSEndpoint.go

```diff
@@ -187,7 +189,15 @@ func (d *ApplicationDNSEndpoint) GetDNSEndpoint() (*externaldns.DNSEndpoint, err
 
 func (d *ApplicationDNSEndpoint) GetExternalTargets(host string) (targets Targets) {
 	targets = NewTargets()
-	for tag, cluster := range d.config.DelegationZones.GetExternalClusterNSNamesByHostname(host) {
+	gt, err := geotags.GeoTag(d.config).GetExternalClusterNSNamesByHostname(host)
```

Comment:
We need to pass `local` tag so we return only external ones

Reply:
it's there we pass whole *config so we know the current geotag